python-manual/modules/ROOT/pages/query-simple.adoc

```diff
@@ -0,0 +1,251 @@
+    records = list(result)  # a list of Record objects
+    summary = result.consume()
+    return records, summary
+
+with driver.session(database="neo4j") as session:
+    records, summary = session.execute_read(get_people)
+
+    # Summary information
+    print("The query `{query}` returned {records_count} records in {time} ms.".format(
+        query=summary.query, records_count=len(records),
+        time=summary.result_available_after
+    ))
+
+    # Loop through results and do something with them
+    for person in records:
+        print(person.data())  # obtain record as dict
+----
+
+where `records` contains the actual result as a list of link:{neo4j-docs-base-uri}/api/python-driver/current/api.html#neo4j.Record[`Record`] objects, and `summary` is a link:{neo4j-docs-base-uri}/api/python-driver/current/api.html#resultsummary[`ResultSummary`] object, containing a summary of execution from the server.
+
+
+== Update the database
+
+To update a node's information in the database, use `MATCH` together with the link:{neo4j-docs-base-uri}/cypher-manual/current/clauses/set/[`SET`] clause in a `Session.execute_write()` call:
+
+.Update node `Alice` to add an `age` property
+[source, python]
+----
+def update_person(tx, name, age):
+    result = tx.run("""
+        MATCH (p:Person {name: $name})
+        SET p.age = $age
+        """, name=name, age=age
+    )
+    summary = result.consume()
+    return summary
+
+with driver.session(database="neo4j") as session:
+    summary = session.execute_write(update_person, name="Alice", age=42)
+    print(f"Query counters: {summary.counters}.")
+----
+
+To create new nodes and relationships linking it to an already existing node, use a combination of `MATCH` and `MERGE` in a `Session.execute_write()` call:
+
+.Create a relationship `KNOWS` between `Alice` and `Bob`
+[source, python]
+----
+def update_person(tx, name, friend):
+    result = tx.run("""
+        MATCH (p:Person {name: $name})
+        MERGE (friend:Person {name: $friend_name})
+        MERGE (p)-[:KNOWS]->(friend)
+        """, name="Alice", friend_name="Bob"
+    )
+    summary = result.consume()
+    return summary
+
+with driver.session(database="neo4j") as session:
+    summary = session.execute_write(update_person,
+                                    name="Alice", friend="Bob")
+    print(f"Query counters: {summary.counters}.")
+----
+
+[WARNING]
+====
+It might feel tempting to create new relationships with a single `MERGE` clause, such as: +
+`MERGE (:Person {name: "Alice"})-[KNOWS]->(:Person {name: "Bob"})`. +
+However, this would result in the creation of an _extra_ `Alice` node, so that you would end up with unintended duplicate records. To avoid this, always `MATCH` the elements that you want to update, and use the resulting reference in the `MERGE` clause (as shown in the previous example). See link:https://neo4j.com/developer/kb/understanding-how-merge-works/[Understanding how MERGE works].
```

Comment:
```suggestion
However, this would result in the creation of an _extra_ `Alice` node, so that you would end up with unintended duplicate records. 
To avoid this, always `MATCH` the elements that you want to update, and use the resulting reference in the `MERGE` clause (as shown in the previous example). 
See link:https://neo4j.com/developer/kb/understanding-how-merge-works/[Understanding how MERGE works].
```